source/renderer/app/api/api.js

```diff
@@ -167,24 +167,24 @@ export default class AdaApi {
     Logger.debug('AdaApi::getWallets called');
     try {
       const response: AdaWallets = await getWallets(this.config);
-      Logger.debug(`AdaApi::getWallets success: ${stringifyData(response)}`);
+      Logger.debug('AdaApi::getWallets success', { wallets: response });
       return response.map(data => _createWalletFromServerData(data));
     } catch (error) {
-      Logger.error(`AdaApi::getWallets error: ${stringifyData(error)}`);
+      Logger.error('AdaApi::getWallets error', { error: `${stringifyError(error)}` });
```

Comment:
@MarcusHurney API now returns properly formatted JSON errors. We should log those without stringifying!